packages/backend/package.json

```diff
@@ -8,22 +8,26 @@
     "@types/cors": "^2.8.13",
     "bcrypt": "^5.1.0",
     "bcryptjs": "^2.4.3",
```

Comment:
Do we need both `bcrypt` and `bcryptjs` packages?

Reply:
no, we don't.